Zoocha/climateweeknyc
# TCG-137: Ability to add blank sponsors.

https://zoocha.atlassian.net/browse/TCG-137

Climate Group wants to be able to add blank sponsors between normal sponsors. For this, we created a new field called "Blank Sponsor?" which hides image/sponsorship fields and adds a new class in `/sponsors` and homepage sponsors block to manage the blank space and borders.

Manual changes:

- Create `field_blank_sponsor`.
- Move it below title field in form display.
- Set image link, sponsor image and sponsorship category fields to optional.
- Add blank sponsor field to view Sponsor page and block.

Test: 

1. Add a few new sponsors and tick "Blank sponsor?" field. Fill title and name (can be the same) and save.
2. Go to `/admin/content/sponsors` and move the blank sponsors wherever you want to place them.
3. Go to Homepage or `/sponsors` page and check the blank sponsors are correctly placed, with no borders.
